XamarinCommunityToolkit/Views/StateLayout/State.shared.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Xamarin.CommunityToolkit.UI.Views
+{
+	public enum State
```

Comment:
Hmm, LayoutState?

State is too general, I believe

Reply:
Thing is, that would mean we have `StateLayout` and `LayoutState`. That confuses me even more 😅

Reply:
Yeah, that's fair. But if we will add several new controls which have "State" property, it will confuse not less :)

Reply:
Agree. Perhaps I shall think a bit longer on an alternative :)

Reply:
Are we still going to change this name?

Reply:
Open to suggestions?